trello/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for trello project.
+
+Generated by 'django-admin startproject' using Django 3.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-r(i@i*pd)iw2cni*$v4yzm&1k#4n_vb32it)znzr5o_k-i+_@s'
```

Comment:
SECURITY WARNING: keep the secret key used in production secret!

Reply:
done